datarobot_provider/example_dags/model_compute_prediction_explanations_dag.py

```diff
@@ -0,0 +1,167 @@
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+from datetime import datetime
+
+import datarobot as dr
+from airflow.decorators import dag
+from airflow.decorators import task
+
+from datarobot_provider.hooks.datarobot import DataRobotHook
+from datarobot_provider.operators.model_insights import ComputeFeatureImpactOperator
+from datarobot_provider.operators.model_predictions import (
+    AddExternalDatasetOperator,
+    RequestModelPredictionsOperator,
+)
+from datarobot_provider.operators.prediction_explanations import (
+    ComputePredictionExplanationsOperator,
+    PredictionExplanationsInitializationOperator,
+)
+from datarobot_provider.sensors.model_insights import DataRobotJobSensor
+
+"""
+Example DAG to create prediction explanations for the specified dataset.
+In order to create Prediction Explanations for a particular model and dataset, you must use the next steps:
+ - Compute feature impact for the model using ComputeFeatureImpactOperator
+ - Initialize prediction explanations for the model using PredictionExplanationsInitializationOperator
+ - Compute predictions for the model and dataset using RequestModelPredictionsOperator
+ - Compute predictions explanations for the model and dataset using ComputePredictionExplanationsOperator
+"""
+
+
+@dag(
+    schedule=None,
+    start_date=datetime(2023, 1, 1),
+    tags=['example', 'dataset', 'model'],
+    # Default json config example:
+    params={'threshold_high': 0.9, 'threshold_low': 0.1, 'max_explanations': 3},
+)
+def compute_model_prediction_explanations(
+    project_id=None,
+    model_id=None,
+    dataset_id=None,
+):
+    if not project_id:
+        raise ValueError("Invalid or missing `project_id` value")
+    if not model_id:
+        raise ValueError("Invalid or missing `model_id` value")
+    if not dataset_id:
+        raise ValueError("Invalid or missing `dataset_id` value")
+
+    add_external_dataset_op = AddExternalDatasetOperator(
+        task_id="add_external_dataset",
+        project_id=project_id,
+        dataset_id=dataset_id,
+    )
+
+    compute_feature_impact_op = ComputeFeatureImpactOperator(
+        task_id="compute_feature_impact",
+        project_id=project_id,
+        model_id=model_id,
+    )
+
+    feature_impact_complete_sensor = DataRobotJobSensor(
+        task_id="feature_impact_complete",
+        project_id=project_id,
+        job_id=compute_feature_impact_op.output,
+        poke_interval=5,
+        timeout=3600,
+    )
+
+    request_model_predictions_op = RequestModelPredictionsOperator(
+        task_id="request_model_predictions",
+        project_id=project_id,
+        model_id=model_id,
+        external_dataset_id=add_external_dataset_op.output,
+    )
+
+    model_predictions_sensor = DataRobotJobSensor(
+        task_id="model_predictions_complete",
+        project_id=project_id,
+        job_id=request_model_predictions_op.output,
+        poke_interval=5,
+        timeout=3600,
+    )
```

Comment:
in the comment above, you mention that PE Initialization should go prior to prediction requests.